test-engineering/load-tests/load_test_setup.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -eu
+
+#Setup environment
+GCLOUD=$(which gcloud)
+SED=$(which sed)
+KUBECTL=$(which kubectl)
+
+DIRECTORY=$(pwd)
+SDK_DIRECTORY=google-cloud-sdk
```

Comment:
This doesn't seem to be used in the script.

Reply:
```suggestion

```